analytic_engine/src/sst/parquet/writer.rs

```diff
@@ -139,12 +142,14 @@ impl RecordBatchGroupWriter {
         Ok(curr_row_group)
     }
 
-    /// Build the parquet filter for the given `row_group`, and then update it
-    /// into `self.parquet_filter`.
-    fn update_parquet_filter(&mut self, row_group_batch: &[RecordBatchWithKey]) -> Result<()> {
+    /// Build the parquet filter for the given `row_group`
```

Comment:
/// Build the parquet filter for the given `row_group`.

Reply:
Isn't same with before?

Reply:
`.` is added.